umh-core/pkg/service/benthos/benthos_backend_test.go

```
@@ -0,0 +1,171 @@
+	It("registers the benthos_monitor worker type through benthos.go's production import", func() {
+		// The fsmv2 worker self-registers on import; benthos.go imports it (line 44)
+		// and constructs the adapter manager under USE_FSMV2_BENTHOS_MONITOR. This
+		// spec lives in package benthos, so the production import graph — not a
+		// test file's own import — is what must have registered the worker. If a
+		// refactor removes the production constructor call (or the import), FF-on
+		// ships nothing while the CPU rig counts a fake win; this gate goes red.
+		registered := false
+		for _, wt := range factory.ListRegisteredTypes() {
+			if wt == fsmv2benthosmonitor.WorkerType {
+				registered = true
+				break
```

golangci-lint (2.12.2)

[error] break with no blank line before

(nlreturn)